DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/ISpeckleObjectToHost/ArcToHostConverter.cs

```diff
@@ -19,11 +19,10 @@ public CurveToHostConverter(IRawConversion<SOG.Point, ACG.MapPoint> pointConvert
   public ACG.Polyline RawConvert(SOG.Arc target)
   {
     // Determine the number of vertices to create along the arc
-    int numVertices = Math.Max((int)target.length, 3); // Determine based on desired segment length or other criteria
+    int numVertices = Math.Max((int)target.length, 50); // Determine based on desired segment length or other criteria
```

Comment:
What are these MAX values? where do they come from?

Reply:
that's a fairly random assignment, we talked with Alan that this needs to be changed to the native API methods later to avoid precision errors

Reply:
Yep... none of this should stay, added another comment in my review (i think on the Circle class)